llvm/test/CodeGen/AArch64/avoid-pre-trunc.ll

```diff
@@ -1,75 +1,36 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
 ; RUN: llc -mtriple=aarch64 < %s | FileCheck %s
 
-define i32 @lower_lshr(<4 x i32> %a, <4 x i32> %b, <4 x i32> %c, <4 x i32> %d, <4 x i32> %e, <4 x i32> %f, <4 x i32> %g, <4 x i32> %h) {
```

Comment:
I think this test case belongs and is covered in the @combine_add_8xi32 function in concat-vector-add-combine.ll so I removed it from this file.